app/views/maps/view_map.html.erb

```diff
@@ -0,0 +1,104 @@
+          <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('balloon1.png') %>">
+          <h3>Make your own map </h3>
+          <p>Use a kite, balloon, pole or drone to take an aerial photo and tell your own visual story of the place </p>
+          <br> <br>
+          <p>
+          <a class="btn btn-primary btn-lg" href= 'https://store.publiclab.org/collections/mapping-kits'>
+              Get a mapping kit<i class="fa fa-angle-double-right fa-fw" style="font-size:18px;color:white;"></i>
+          </a>
+          <a class="btn btn-primary btn-lg" href="<%= new_map_url %>">
+              Add a new map <i class="fa fa-plus fa-fw" style="font-size:12px;color:white;"></i>
+          </a>
+          </p>
+
+      </div>
+  </div>
+</div>
+  <br><br>
+<div class="all-maps light-blue">
+    <br>
+    <h3 class="text-center"> Nearby Maps </h3>
+         <hr style="max-width: 500px;">
+    <br>
+    <%= render :partial => 'front_ui/maps', :locals => { :maps => @maps } %>
+</div>
+  <br><br>
+
+
+  <div class="container-fluid">
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('quiz.png') %>">
+              <br>
+              <h3>Ask a question</h3>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
+              <img class="mx-auto d-block faded" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('binoculars.png') %>">
+              <br>
+              <h3>Projects/ Requests for observations</h3>
+          </div>
+      </div>
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> Get help from others and share ideas with the PublicLab Community</p>
+              <br>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> Community observations and monitoring are a key source of environmental data. Help add to one of the [projects] here. Read more or post your oen call for input</p>
+              <br>
+          </div>
+      </div>
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> <%= link_to "Ask a question", 'https://publiclab.org/questions/new', target: 'blank', class: "btn btn-lg btn-primary" %></p>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
+              <p> <%= link_to "Post an Observation", 'https://publiclab.org/post', target: 'blank', class: "btn btn-outline-secondary btn-lg" %></p>
+
+          </div>
+      </div>
+
+  <script>
+      var map
+      (function(){
+      window.mapKnitter = new MapKnitter.Map({
+        latlng:     L.latLng(<%= @map.lat %>, <%= @map.lon %>),
+        zoom:       <%= @map.zoom %>,
+        readOnly:   true,
```

Comment:
Is this working yet? It had gotten a bit broken in all the recent changes. We may need to coordinate with @sashadev-sky and maybe @ViditChitkara on https://github.com/publiclab/Leaflet.DistortableImage/issues/351 -- ah, i see solution in https://github.com/publiclab/mapknitter/pull/892 we can review! Looking... in a bit!

Reply:
@jywarren @ViditChitkara @cesswairimu I merged the fix this should be good to go now!